src/test/suite/runSingleRecordedTest.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Add the file path ending of a recorded test and ONLY that one will run
+ */
+
+// example:
+// const filenameEnd = "textual/takePairRound.yml";
+
+const filenameEnd = "";
```

Comment:
This setup is begging for trouble imo.  If someone accidentally leaves a filename here and we miss it, they will have deactivated our entire test suite.  I'd prefer having a special npm command for this one. Can we not just use [Mocha's `--grep`](https://mochajs.org/#-grep-regexp-g-regexp) feature? We could also try adding a npm arg and then passing it via env var or something

Reply:
Any of these options would be super helpful from a contributor's perspective. The suite runs fast but not fast enough for a good feedback loop.